app/cli/internal/telemetry/posthog/posthog_test.go

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright 2024 The Chainloop Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package posthog_test
+
+import (
+	"testing"
+
+	"github.com/chainloop-dev/chainloop/app/cli/internal/telemetry/posthog"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewClient(t *testing.T) {
```

Comment:
This could probably use some table driven tests, with the different combinations + happy path?

Reply:
Oki